jobs.groovy

```diff
@@ -692,12 +696,31 @@ branches.each {
         for (job_entry in job_type_entry.value) {
             job_name = job_entry.key
             job_values = job_entry.value
-            define_job(dashBranchName, branchName, job_type, job_name, job_values)
+            define_job(dashBranchName, branchName, job_type, job_name, job_values, isReleaseBuild)
         }
     }
 
     // create multijob that aggregates the individual jobs
-    build_multijob(dashBranchName, branchName)
+    build_multijob(dashBranchName, branchName, isReleaseBuild)
+}
+
+
+/* --------------------
+   MAIN ACTION:
+   --------------------
+*/
+// Iterate over every branch, and create folders, jobs
+branches.each {
+    println("iterating over branch... ${it}")
+    branchName = it
+    dashBranchName = branchName.replace("/","-")
+    isReleaseBuild = branchName.startsWith("target-branch-")
```

Comment:
Where does the "target-branch-" magic string come from? Please add a comment explaining why.